RxCocoa/OSX/Proxies/RxTextFieldDelegateProxy.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  RxTextFieldDelegateProxy.swift
+//  Rx
+//
+//  Created by Junior B. on 21/06/16.
+//  Copyright © 2016 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+import Cocoa
+#if !RX_NO_MODULE
+import RxSwift
+#endif
+
+/**
+ Delegate proxy for `NSTextField`.
+ 
+ For more information take a look at `DelegateProxyType`.
+ */
+public class RxTextFieldDelegateProxy
+    : DelegateProxy
+    , NSTextFieldDelegate
+, DelegateProxyType {
+    
+    public let textSubject = PublishSubject<String>()
+    
+    /**
+     Typed parent object.
+     */
+    public weak private(set) var textField: NSTextField?
+    
+    /**
+     Initializes `RxTextFieldDelegateProxy`
+     
+     - parameter parentObject: Parent object for delegate proxy.
+     */
+    public required init(parentObject: AnyObject) {
+        self.textField = (parentObject as! NSTextField)
+        super.init(parentObject: parentObject)
+    }
+    
+    // MARK: Delegate methods
+    
+    public override func controlTextDidChange(notification: NSNotification) {
+        let textField = notification.object as! NSTextField
+        let nextValue = textField.stringValue
+        self.textSubject.on(.Next(nextValue))
```

Comment:
use `as?` to avoid crash?
you don't need `self` here
remove extraneous `nextValue`?
```swift
let textField = notification.object as? NSTextField
textSubject.on(.Next(textField.stringValue))
```

or 

```swift
if let textField = notification.object as? NSTextField {
    textSubject.on(.Next(textField.stringValue))
}
```

Depends on if you want to fire on nil strings like `textSubject.on(.Next(nil))` or not